imagetest/test_suites/image_validation/image_validation_test.go

```diff
@@ -64,14 +64,21 @@ func TestFQDN(t *testing.T) {
 	}
 
 	// This command is not safe on multi-NIC VMs. See HOSTNAME(1), section 'THE FQDN'.
-	cmd := exec.Command("/bin/hostname", "-A")
+	cmd := exec.Command("/bin/hostname", "-a")
 	out, err := cmd.Output()
 	if err != nil {
 		t.Fatalf("hostname command failed")
 	}
 	hostname := strings.TrimRight(string(out), " \n")
 
-	if hostname != metadataHostname {
+	cmd = exec.Command("/bin/hostname", "-d")
```

Comment:
please read the hostname manpage. this is a not-recommended option. what are you trying to do? let's discuss offline

Reply:
The rhel 7-4 sap will return blank for `hostname -A`. Let's discuss offline.

Reply:
Revert this as this is not right solution to fix bug.